README.md

```diff
@@ -1,4 +1,3 @@
-
 # HTTP Request/Response Cycle - Lab
```

Comment:
I'm adding comments on the README rather than on the notebook because the interface is better, but the idea is that these requested changes would happen in the notebook then be exported to the README using `dscreate`